tools/toolchain/dbuild

```diff
@@ -187,8 +187,10 @@ docker_common_args+=(
        -v "$tmpdir:/tmp" \
        -v "$MAVEN_LOCAL_REPO:$MAVEN_LOCAL_REPO" \
        -v /etc/localtime:/etc/localtime:ro \
+       -v "$HOME/.ccache:$HOME/.ccache" \
        -w "$PWD" \
        -e HOME="$HOME" \
+       -e PATH="/usr/lib64/ccache:/usr/local/bin:/usr/sbin:/usr/bin" \
```

Comment:
Tricks like that often come back to bite us later, e.g. Fedora-42 (if it ever happens) adds something meaningful to `/usr/local/sbin` that's missing in this list and :boom: -- we're dead

Reply:
Why not the customary `PATH=/usr/lib64/ccache:$PATH`?

Reply:
> Tricks like that often come back to bite us later, e.g. Fedora-42 (if it ever happens) adds something meaningful to /usr/local/sbin that's missing in this list and boom -- we're dead

@xemul agree that the PATH might be changed in the future but we're on the safe side since dbuild image changes going through the CI and should catch such critical changes

Reply:
> Why not the customary PATH=/usr/lib64/ccache:$PATH?

@psarna last time I checked, it didn't work with Dockerfile, the PATH will be the host's PATH, same for ${PATH}